### DISCUSSION TITLE
backup repos with migrations vs github-backup

### DISCUSSION THREAD

**Discussion by pammecrandall** [community/community#38514] [New to GitHub]
I'm considering using either the github migrations api or https://pypi.org/project/github-backup to periodically save our repos and their metadata and I'm wondering if anyone has done this comparison already and has any recommendations.  It looks like they capture basically the same data except that github-backup is able to clone LFS repositories and the migrations api isn't.

**Comment by ossanna16** [comment]
Hi there @pammecrandall and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by pammecrandall** [reply]
Hi!  I will take a look.  Thank you!
